### PR TITLE
Add Theme Kit, Twine and Turbograft repos

### PR DESCRIPTION
**Pull request by cshold**
Add in Theme Kit, Twine, and Turbograft, as they're both public and open but not listed. Any reason not to showcase any of these?

cc @m-ux @tylerball

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,4 +113,3 @@ <h1 class="hero-text">As a team that greatly benefits from open-source software,
 <script>
   var repos = {{ site.github.public_repositories | jsonify }};
 </script>
-
```
Comment (cshold) [inline index.html]: This triggers a rebuilt of GitHub's Jekyll data

**Comment by m-ux** [comment]
Hmmm... I definitely don't have enough context to know why these wouldn't be listed here, but imagine there's a reason.

**Comment by cshold** [comment]
@pushrax @TheMallen @qq99 thoughts on adding these to our open source listings?

**Comment by pushrax** [comment]
Seems reasonable.

**Pull request merged**